guides/source/active_record_validations.md

```diff
@@ -777,28 +847,32 @@ class Person < ApplicationRecord
 end
 ```
 
-WARNING. Note that some databases are configured to perform case-insensitive
-searches anyway.
+WARNING: Some databases are configured to perform case-insensitive searches
+anyway.
 
-There is a `:conditions` option that you can specify additional conditions as a
-`WHERE` SQL fragment to limit the uniqueness constraint lookup (e.g.
-`conditions: -> { where(status: 'active') }`).
+A `:conditions` option can be used to specify additional conditions as a `WHERE`
+SQL fragment to limit the uniqueness constraint lookup (e.g. `conditions: -> {
+where(status: 'active') }`).
```

Comment:
I think the example here is a bit more readable if we use the triple backticks and a full example:

````suggestion
A `:conditions` option can be used to specify additional conditions as a `WHERE`
SQL fragment to limit the uniqueness constraint lookup:

```ruby
validates :name, uniqueness: { conditions: -> { where(status: 'active') } }
```
````